workers/scanner.go

```diff
@@ -399,6 +425,36 @@ func (s *Scanner) managedRefreshMinPinners() {
 	s.mu.Unlock()
 }
 
+// staticEligibleToPin returns true when this server is either pinning less than
+// AlwaysPinThreshold data or it's in the lower PinningRangeThresholdPercent
+// section of servers when ordered by contract data in descending order.
+// A server is always eligible if it's last in the list.
+func (s *Scanner) staticEligibleToPin(ctx context.Context) (bool, error) {
+	pinnedData, err := s.staticDB.ServerLoad(ctx, s.staticServerName)
```

Comment:
tip: You already wrote a good test so there is no action item here but I would probably have written a `isEligibleToPin(pinnecData, pos, total)` function which is called by `scanner.staticEligibleToPin`.
Unit testing a function like that is super easy to do with a table test and then you only need to write a quick test for the scanner method which verifies that it calls the function.

Reply:
*facepalm*
Yes... Yes, that makes a lot of sense. :)
I won't change this one due to time pressure but that's a great tip. Thanks!